### ISSUE TITLE
Firefox?

### ISSUE THREAD

**Issue by Mioni** [rocketshipapps/adblockfast#23]
Any chance to make this extension in Firefox?
Thank you.

**Comment by oldestlivingboy** [comment]
Yes :-), as soon as [Mozilla’s new extension API](https://wiki.mozilla.org/WebExtensions) is in production (ETA: “a few more months” according to [their FAQ](https://wiki.mozilla.org/WebExtensions/FAQ)).

**Comment by oldestlivingboy** [comment]
Reopening to track progress – the latest (first release is in Firefox 45 in March): https://blog.mozilla.org/addons/2015/12/21/webextensions-in-firefox-45-2/ …

**Comment by E3V3A** [comment]
Yeah, great to hear. How can I help and get started in supporting this?

How about Dolphin browser?

**Comment by TPS** [comment]
Based on https://github.com/rocketshipapps/adblockfast/blob/master/android/app/src/main/res/raw/blocked.txt, isn't https://addons.mozilla.org/addon/bluhell-firewall/ the Ff version, anyway?

**Comment by bric3** [comment]
any update ?

**Comment by codeduph** [comment]
Well, I really like "Bluhell Firewall" on FF but I felt like it wasn't maintained properly as I see some annoying ads as the months go by.

So, yeah! any update for FF users?

**Comment by s0kil** [comment]
Firefox support should be simple now, https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Porting_a_Google_Chrome_extension

**Comment by s0kil** [comment]
I have tested the extension using [web-ext](https://github.com/mozilla/web-ext), and it appears to be working properly on Firefox

**Comment by oldestlivingboy** [comment]
@s0kil Awesome, thank you for keeping tabs and for testing<em>!</em> We’ll aim to submit to addons.mozilla.org in the next week.

**Comment by Dentrax** [comment]
Hey, any updates? I can't wait!
